modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/NodeJSServerCodegen.java

```diff
@@ -314,4 +314,10 @@ public void serialize(Double val, JsonGenerator jgen,
         }
         return super.postProcessSupportingFileData(objs);
     }
+
+    @Override
+    public String removeNonNameElementToCamelCase(String name) {
```

Comment:
would be nice to add a docstring explaining what the function does, parameter description, etc